crates/key-value-azure/src/store.rs

```diff
@@ -353,15 +412,18 @@ impl Cas for CompareAndSwap {
 }
 
 impl AzureCosmosStore {
-    async fn get_pair(&self, key: &str) -> Result<Option<Pair>, Error> {
+    async fn get_pair<F>(&self, key: &str) -> Result<Option<F>, Error>
```

Comment:
Should we change the name of this function?  It seems like it's no longer limited to pairs.

Reply:
Maybe `get_entity`?

Reply:
updated to `get_entity`